src/containers/Dev/Snapshots.tsx

```diff
@@ -161,8 +185,7 @@ const Snapshots: React.FC = () => {
                   />
                 </>
               ) : null} */}
```

Comment:
Can all the code above not in use be deleted?

Reply:
I dunno, this is all stuff that Andrei made and commented out before I even looked at it, so I didn't really take much notice of it. You reckon just get rid of it?

Reply:
Hmmm this is the diff stuff! Yes, I guess we should leave it for now.